universal-application-tool-0.0.1/app/services/program/ProgramServiceImpl.java

```diff
@@ -113,7 +115,11 @@ public ProgramDefinition getProgramDefinition(long id) throws ProgramNotFoundExc
       return ErrorAnd.error(errors);
     }
     Program program =
-        programDefinition.toBuilder().setName(name).setDescription(description).build().toProgram();
+        programDefinition.toBuilder()
+            .setLocalizedName(ImmutableMap.of(Locale.US, name))
```

Comment:
Thought (blocking): this would not update the admin description, which may not be what we want?

Reply:
That's a good point - I'm not sure what the behavior should be for now. Eventually, these name/descriptions will be treated separately - adminName will always be immutable, but the others can change. I will leave this for now but will update in #777